spec/fixtures/test_module/manifests/tag_notice.pp

```diff
@@ -0,0 +1,5 @@
+class tag_parameter_test ($tag=undef){
+  notify { 'should should pass':
```

Comment:
Unless I've misunderstood the intention, I think this should be 'should pass', rather than 'should should pass'. What do you think?